### PR TITLE
Add karma test

### PR DESCRIPTION
**Pull request by Minglee01**
@incuna/js 

repo is now ready to have karma tests.

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -29,16 +36,52 @@ module.exports = function (grunt) {
                 config: '.jscsrc'
             },
             src: '<%= config.files.lint %>'
-        }
+        },
```
Comment (wytrych) [inline Gruntfile.js]: stray comma

File: Gruntfile.js [hunk 2]
```diff
@@ -2,18 +2,25 @@
...
-    grunt.loadNpmTasks('grunt-jscs');
+    require('time-grunt')(grunt);
 
     grunt.initConfig({
         config: {
+            lib: 'bower_components',
+            modules: 'src',
             files: {
                 lint: [
                     'src/**/*.js'
```
Comment (wytrych) [inline Gruntfile.js]: I think the lint files should also contain `karmaTests` and the `Gruntfile`

Comment (Minglee01) [inline Gruntfile.js]: not aware of any other repo that lints the gruntfile

File: Gruntfile.js (hunk 1 above)
Comment (henrahmagix) [inline Gruntfile.js]: You can have the linters lint this file to catch these

File: Gruntfile.js (hunk 2 above)
Comment (wytrych) [inline Gruntfile.js]: https://github.com/incuna/teva-becon/pull/435

File: Gruntfile.js (hunk 1 above)
Comment (Minglee01) [inline Gruntfile.js]: getting erros like ```
ReferenceError: module is not defined in /Users/minglee/Sites/djangular-rest-framework/tests/unit/test.js (line 10)
```
 not sure what the fix would be
looking at another of our repose i cant seem to find the defination.

File: Gruntfile.js (hunk 2 above)
Comment (henrahmagix) [inline Gruntfile.js]: It's just another js file, so it should be linted. I've added it to the linters on projects I've been on

**Comment by Minglee01** [comment]
@wytrych  i think ive addressed everything now please rereview. thanks

**Comment by wytrych** [comment]
Cool, HWTI

**Pull request merged**